index.html

```diff
@@ -768,6 +768,87 @@ <h2>Verifiable Credential</h2>
           <pre data-include="./examples/vc-2.0/verifiable-credential.jwt" data-include-format="text"></pre>
         </aside>
       </section>
+
+    <section>
+      <h2>Mapping to the Verifiable Credentials Data Model</h2>
+      <p>
+We define the following mapping from <code>vc+jwt</code> to
+<code>credential+ld+json</code>:
+      </p>
+      <ul>
+        <li>
+Extract <code>iss</code>, <code>sub</code>, <code>iat</code>, <code>nbf</code>,
+<code>exp</code>, <code>jti</code>, and <code>aud</code> as registered claims.
+        </li>
+        <li>
+Set aside all other claims as subject claims.
+        </li>
+        <li>
+To construct the VCDM-complaint JSON:
+          <ul>
+            <li>
+Set the value of <code>@context</code> to be
+<code>"https://www.w3.org/ns/credentials/v2"</code>.
+            </li>
+            <li>
+Set the value of <code>type</code> to be <code>["VerifiableCredential"]</code>.
+            </li>
+            <li>
+Set the value of the issuer property to one of the following:
+            <ul>
+              <li>
+If the value of <code>iss</code> is a URL, use the value of <code>iss</code>.
+              </li>
+              <li>
+If the value of <code>iss</code> is not a URL, use the concatenation of "urn:"
+and the value of <code>iss</code>.
+              </li>
+            </ul>
+            </li>
+            <li>
+If <code>iat</code> is present, set the value of <code>id</code> as the
```

Comment:
```suggestion
If <code>jti</code> is present, set the value of <code>id</code> as the
```

I believe this was a typo and you wanted to use `jti` for the `id` of the VC.

Reply:
indeed.

Reply:
changed in [e7b8855](https://github.com/w3c/vc-jwt/pull/60/commits/e7b8855702f1cfd378cea2df3c7f3f0223f3f608)